docs/blog/moving-portland-prague-2020-online.rst

```diff
@@ -0,0 +1,52 @@
+.. post:: Apr 29, 2020
+   :tags: portland-2020, prague-2020, conferences, covid-19
+
+Moving our 2020 conferences online
+==================================
+
+Hey everyone,
+
+We hope that you're holding up during these trying times.
+We are all stuck at home, but still thinking about our docs community :)
+
+**We've concluded that it's not possible for us to hold our Portland or Prague conferences in person this year.**
+Safety of the community and everyone else comes first.
+
+**We're planning to hold an online event, with a new format, sometime in August**.
+We will be releasing additional details as we confirm them.
+
+If you currently have a ticket to Write the Docs Portland, your options are:
+
+- pre-order a ticket to the online event for $100 and get the rest refunded.
+- pre-order a ticket to the online event for $100 and contribute some of the rest to sustaining the organization.
+- get a full refund on **May 10th, 2020**. This happen automatically if you don't select either of the previous options.
+
+`Let us know`_ which option you'd like.
+
+We'll be reaching out to all of our sponsors to talk about moving their sponsorship to our online event, and we are excited about offering new sponsorship benefits provided by the new format.
+
+.. _Let us know: http://TODO
+
+Thoughts on the new format
+--------------------------
+
+We're looking at ways to keep the spirit of our events, in an online format, still providing a gathering place for the community.
+
+* Writing Day, where the community works on projects and learns from each other. These will work well remotely.
+* Unconference, where our community has discussions and shares ideas. We think our Slack network already approximates this pretty well, but we'll be adding a little extra structure and moderation during the event to help the discussions move along.
+* Presentations, where we share ideas, insights, and best practices. We're investigating live and prerecorded options, with curated and real time Q&A with speakers and other experts.
+* Job Fair, matching attendees to sponsors and jobs. We'll be building on our Job Board, offering our sponsors the ability to showcase the work they're doing and the roles they're hiring for.
+
+We haven't found good ways to do our Hike or Boat Ride online, so you'll just have to enjoy a small bit of your local woods or water, at socially responsible distances, on our behalf.
+
+Benefits that come from the new reality
+---------------------------------------
+
+We are sad not to be able to have our upcoming events in person,
+but we also want to note that there are some benefits to moving online.
+People who haven't been able to travel to our events will still be able to speak and attend, making the event more accessible to everyone.
```

Comment:
```suggestion
The biggest benefit is accessibility to the event itself, since people who haven't been able to travel to our events will be able to speak and attend the virtual events, making the events more accessible to everyone. The lower ticket price will also help more people from all walks of life attend the event.
```